libs/blocks/merch-card/merch-card.js

```diff
@@ -340,6 +340,21 @@ const setMiniCompareOfferSlot = (merchCard, offers) => {
   merchCard.appendChild(miniCompareOffers);
 };
 
+const updateBigPrices = (merchCard) => {
+  const prices = merchCard.querySelectorAll('strong > em > span[is="inline-price"]');
+  const isMobile = window.matchMedia('(max-width: 1199px)').matches;
+  prices.forEach((span) => {
+    const strongTag = span.parentNode.parentNode;
+    const emTag = span.parentNode;
+    strongTag.replaceChild(span, emTag);
+    if (!isMobile) {
+      span.style.cssText = 'font-size: 24px; line-height: 22.5px;';
```

Comment:
why not using classes? and something like
`span.classList.add(isMobile ? 'big-price-mobile' : 'big-price');`

Reply:
I did that initially but @yesil suggested direct style instead to avoid performance impact